### PR TITLE
Fix #rrggbbaa color to rgba for better browser compatibility

### LINKED ISSUE #4988: Wrong LESS conversion
- Gitea version (or commit ref): master (31d54880598beb7fdc9832147521063292ae9c0c)

public/css/index.css contains style like:
```
background:#21BA4518 18
```
or
```
border-left:1px solid #21BA4580 80
```

They are not valid CSS syntax.
Maybe LESS.js (3.6.0) generates wrong code for rgba value.

### PR DESCRIPTION
**Pull request by SagePtr**
Fixes #4988

Color format #rrggbbaa is not widely supported by browsers yet: https://caniuse.com/#search=rgba
Less [fade](http://lesscss.org/functions/#color-operations-fade) function is better here as it generates rgba(r, g, b, a) format output which has much better browser support. Also Less <3.8.1 handles #rrggbbaa format wrong as pointed in #4988 issue.

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Review by adelowo — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by techknowlogick** [comment]
Maintainer note: This doesn't have a milestone applied to it. I'd like it to be in 1.6, but we should wait for #4971 to get merged first before merging this as the other PR already had to resolve conflicts arising from other CSS changes. I don't want to have the other contributor to have to resolve the same conflicts again.

That being said, I am only one of many maintainers and so if someone has a different opinion that is fine too.

**Comment by lafriks** [comment]
Please rebase and resolve conflicts

**Comment by SagePtr** [comment]
Done it and regenerated stylesheet

**Pull request merged**
